plugin/src/main/groovy/com/btkelly/gnag/extensions/GitHubExtension.java

```diff
@@ -32,6 +33,14 @@ public GitHubExtension(Project project) {
         this.project = project;
     }
 
+    public void rootUrl(String rootUrl) {
+        this.rootUrl = rootUrl;
+    }
+
+    public String getRootUrl() {
+        return project.hasProperty("rootUrl") ? (String) project.property("rootUrl") : rootUrl != null ? rootUrl : "https://api.github.com/repos/";
```

Comment:
This is because it has the default value of `https://api.github.com/repos/`. So if it's not provided by the command line or the config block it will return the default value.